Jenkinsfile

```diff
@@ -72,31 +72,44 @@ pipeline {
                description: 'Priority of this build.  DO NOT USE WITHOUT PERMISSION.')
         string(name: 'TestTag',
                defaultValue: 'pr daily_regression',
-               description: 'Test-tag to use for the Functional Hardware Small/Medium/Large stages of this run (i.e. pr, daily_regression, full_regression, etc.)')
+               description: 'Test-tag to use for the Functional Hardware stages of this run ' +
+                            '(i.e. pr, daily_regression, full_regression, etc.)')
+        string(name: 'TestRepeat',
+               defaultValue: '',
+               description: 'Test-repeat to use for this run.  Specifies the number of times to ' +
+                            'repeat each functional test. CAUTION: only use in combination with ' +
+                            'a reduced number of tests specified with the TestTag parameter.')
         string(name: 'TestProvider',
                defaultValue: 'ucx+dc_x',
-               description: 'Provider to use for the Functional Hardware Small/Medium/Large stages of this run (i.e. ucx+dc_x)')
+               description: 'Provider to use for the Functional Hardware stages of this run (i.e. ucx+dc_x)')
         string(name: 'BaseBranch',
                defaultValue: base_branch,
-               description: 'The base branch to run weekly-testing against (i.e. master, or a PR\'s branch)')
-        booleanParam(name: 'CI_small_TEST',
-                     defaultValue: true,
-                     description: 'Run the CI Functional Hardware Small test stage')
+               description: 'The base branch to run testing against (i.e. master, or a PR\'s branch)')
+        string(name: 'CI_RPM_TEST_VERSION',
+               defaultValue: '',
+               description: 'Package version to use instead of building. example: 1.3.103-1, 1.2-2')
+        string(name: 'CI_PR_REPOS',
+               defaultValue: '',
+               description: 'Repository to include for packages used to the build and test ' +
+                            'nodes, in the project@PR-number[:build] format')
```

Comment:
Similar to https://github.com/daos-stack/daos/pull/10875
https://github.com/daos-stack/daos/pull/10875#discussion_r1036344541